pkg/volume/cephfs/cephfs.go

```diff
@@ -264,6 +287,14 @@ func (cephfsVolume *cephfs) GetPath() string {
 	return cephfsVolume.plugin.host.GetPodVolumeDir(cephfsVolume.podUID, utilstrings.EscapeQualifiedNameForDisk(name), cephfsVolume.volName)
 }
 
+// GateKeyringPath creates cephfuse keyring path
+func (cephfsVolume *cephfs) GetKeyringPath() string {
```

Comment:
this leaks keyring - anybody has access to the directory can see the keyring. 

see https://github.com/kubernetes/kubernetes/blob/release-1.8/pkg/volume/rbd/rbd_util.go#L297

Reply:
I change it, no matter if mount fail, we delete the keyring after execFuseMount